pages/spicedb/concepts/schema.mdx

```diff
@@ -283,8 +286,38 @@ definition document {
 ```
 
 <Callout type="info">
-It is _recommended_ that the right side of all arrows refer to **permissions**, instead
-of relations. This allows for easy nested computation, and is more readable.
+  It is _recommended_ that the right side of all arrows refer to
+  **permissions**, instead of relations. This allows for easy nested
+  computation, and is more readable.
```

Comment:
one sentence per line

Reply:
Formatter keeps resetting it... I'll reword

Reply:
Fixed